tools/devdocs-generator.ps1

```diff
@@ -642,3 +645,14 @@ $indexContent += $(Generate-TypeSectionContent $featureEntries) + "`r`n"
 Set-Content -Path "../docs/devdocs.md" -Value $indexContent -Encoding utf8
 
 Update-Progress "Process Completed" 100
+
+Write-Host "Running Preprocessing"
+
+# Dot source the 'Invoke-Preprocessing' Function from 'tools/Invoke-Preprocessing.ps1' Script
+$preprocessingFilePath = ".\Invoke-Preprocessing.ps1"
+. $preprocessingFilePath
+
+$excludedFiles = @('.\.git\', '.\.gitignore', '.\.gitattributes', '.\.github\CODEOWNERS', '.\LICENSE', ".\tools\Invoke-Preprocessing.ps1", '*.png', '*.exe')
+$msg = "Pre-req: Code Formatting"
+$workingdir = $PSScriptRoot -replace '\\tools$', ''
+Invoke-Preprocessing -WorkingDir "$workingdir" -ExcludedFiles $excludedFiles -ProgressStatusMessage $msg -ThrowExceptionOnEmptyFilesList
```

Comment:
I don't know if this step is necessary, as the point of the pre-processor is to format all the repo (specifically the PS Files) to follow certain conventions... but this's done _after_ the dev docs have been generated. So I don't really see the point of running it inside this script (maybe _before_ reading repo files & generating docs.. but even then, this isn't a concern of this script :/).

If there's a reason behind this... then I'd like to know if you don't mind, thanks in advance 😄

Reply:
I did not work big on that part yet, but i want to launch the preprocessor with the filter to only apply changes on generated files (devdocs & table of content)